# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: widget/cocoa/nsCocoaWindow.h
# Commit: 5791073c28b7
# Full Hash: 5791073c28b727964776360a5b4dfd5c004ce152
# Author: Hiroyuki Ikezoe <hikezoe.birchill@mozilla.com>
# Date: 2021-04-13 03:56:47
# Description:
#   Bug 1703762 - Delegate StartAsyncAutoscroll and StopAsyncAutoscroll calls to the proper widget mPopupContentView. r=botond
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D111575
# ==============================================================================

diff -r 4a25e6b0bf09 -r 5791073c28b7 widget/cocoa/nsCocoaWindow.h
--- a/widget/cocoa/nsCocoaWindow.h	Mon Apr 12 22:50:22 2021 +0000
+++ b/widget/cocoa/nsCocoaWindow.h	Mon Apr 12 23:05:40 2021 +0000
@@ -362,6 +362,10 @@
 
   bool AsyncPanZoomEnabled() const override;
 
+  bool StartAsyncAutoscroll(const ScreenPoint& aAnchorLocation,
+                            const ScrollableLayerGuid& aGuid) override;
+  void StopAsyncAutoscroll(const ScrollableLayerGuid& aGuid) override;
+
  protected:
   virtual ~nsCocoaWindow();
 